clusterloader/docs/design.md

```diff
@@ -185,21 +185,27 @@ In many cases, it will be enough for users for many namespaces to contain the sa
 objects (or to be more specific: objects created from the same templates). Obviously,
 we want the config for the test to be as small as possible.
 As a result, we will introduce the following rules:
-- In the top-level test definition, we will define the number of namespaces that will
-  be used by a test.
-- All the namespaces will have names of a form “namespace-<number>” for number
+- In the top-level test definition, we will define the number of default namespaces
```

Comment:
'number of nmespaces that will be automanaged by clusterloader'?

Reply:
done